lib/streamlit/elements/widgets/button.py

```diff
@@ -279,6 +291,25 @@ def button(
            https://doc-button-icons.streamlit.app/
            height: 220px
 
+        **Example 3: Use keyboard shortcuts**
+
+        The following example shows how to use keyboard shortcuts to trigger a
+        button. If you use any of the platform-dependent modifiers (Ctrl, Cmd,
+        or Mod), they are interpreted interchangeably and always displayed to
+        the user to match their platform.
+
+        >>> import streamlit as st
+        >>>
+        >>> with st.container(horizontal=True, horizontal_alignment="distribute"):
+        >>>     "`A`" if st.button("A", width=100, shortcut="Shift+A") else "` `"
+        >>>     "`S`" if st.button("S", width=100, shortcut="Ctrl+S") else "` `"
+        >>>     "`D`" if st.button("D", width=100, shortcut="Cmd+D") else "` `"
+        >>>     "`F`" if st.button("F", width=100, shortcut="Mod+F") else "` `"
```

Comment:
Maybe we add one example here using three keys: `Mod+Alt+F` to show that this is also possible